code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm

```diff
@@ -103,3 +112,73 @@
 		datum_reference.qliphoth_change(-1)
 	return
 
+/mob/living/simple_animal/hostile/abnormality/judgement_bird/BreachEffect(mob/living/carbon/human/user)
+	..()
+	if(CheckCombat())
+		judgement_damage = 65
+		return
+
+	var/mob/living/simple_animal/hostile/runawaybird/V = new(get_turf(src))
+	birdlist+=V
+	V = new(get_turf(src))
+	birdlist+=V
+	V = new(get_turf(src))
+	birdlist+=V
```

Comment:
Do a `for` loop instead; You also don't need a "V" variable at all.

Reply:
For loops are technically slower.......